aviary/subsystems/mass/gasp_based/fixed.py

```diff
@@ -97,7 +97,7 @@ def setup(self):
         )
 
         self.declare_partials(
-            "c_gear_loc", Aircraft.LandingGear.MAIN_GEAR_LOCATION, val=0)
+            "c_gear_loc", Aircraft.LandingGear.MAIN_GEAR_LOCATION)
```

Comment:
@Kenneth-T-Moore can correct me if I'm wrong, but I think this 0 is important when Aircraft.Design.SMOOTH_MASS_DISCONTINUITIES is False

Reply:
I get an OMDeprecationWarning: Partial was declared to be exactly zero. This is inefficient and the declaration should be removed. In a future version of OpenMDAO this behavior will raise an error.